game/scripts/vscripts/components/boss/bosses.lua

```diff
@@ -8,7 +10,9 @@ Bosses = {
     "npc_dota_creature_temple_guardian_spawner",
     "npc_dota_boss_tier_6"
   },
-  -- tier 1
```

Comment:
leave the tier comments in so it's more clear. "type 2" doesn't imply "tier 1", this file becomes counter-intuitive to new eyes